docs/changelog.rst

```diff
@@ -13,6 +13,8 @@ CHANGELOG
 **New features**
 
 - Add new setting `DIRECTION_ON_LINES_ENABLED` to have the `direction` field on lines instead of blades
+- Partially handle translated fields, when setting ``fill_empty_translated_fields`` is True,
+  all empty translated field will be set with value
```

Comment:
Proposed rephrasing : "Partially handle translated fields: when setting ``fill_empty_translated_fields`` to True,
  all empty translation fields for all languages will be set with the parsed value"
The `django-modeltranslation` lib names the `*_fr`, `*_en` fields _translation fields_. I think it would be nice to keep that to make it clear when we talk about the translated field (for instance `label`) and the translation fields (`label_fr`, `label_en`, etc).